docs/MISSION_AND_VALUES.md

```diff
@@ -0,0 +1,75 @@
+# Mission and Values
+
+This document is meant to help guide decisions about the future of `napari`, be it in terms of
+whether to accept new functionality, changes to the styling of the code or graphical user interface (GUI),
+or whether to take on new dependencies, among other things. It serves as a point of reference for [core developers](CORE_DEV_GUIDE.md) actively working on the project, and an introduction for
+newcomers who want to learn a little more about where the project is going and what the team's
+values are. You can also learn more about how the project is managed by looking at our [governance model](GOVERNANCE).
+
+## Our founding principles
+
+The original napari team came together around a shared vision for a Python based image viewer that was
```

Comment:
```suggestion
The original napari team came together around a shared vision for a Python based image viewer that is
```

Reply:
fixed, i also swapped "original" to "founding" as I think it sounds a bit better